core/include/scipp/core/string.h

```diff
@@ -83,6 +83,8 @@ template <class T> std::string element_to_string(const T &item) {
   using std::to_string;
   if constexpr (std::is_same_v<T, std::string>)
     return {'"' + item + "\", "};
+  else if constexpr (std::is_same_v<T, bool>)
```

Comment:
Maybe [this](https://github.com/scipp/scipp/blob/masking-doc/core/include/scipp/core/string.h#L50) overload is not necessary with this?

Reply:
No, it is using that, so it should be necessary?

Reply:
My bad. I did not see the name of the function 🤦‍♂